runtime/src/main/java/org/corfudb/runtime/collections/FGMap.java

```diff
@@ -26,6 +26,9 @@
         objectType = ObjectType.STATELESS)
 public class FGMap<K, V> extends AbstractCorfuWrapper<FGMap<K,V>> implements Map<K, V> {
 
+    @Getter
+    private ISMRMap.SMRMapGarbageManager<K> mapGarbageManager = new ISMRMap.SMRMapGarbageManager<>();
```

Comment:
I don't think FGMap needs this. This class is effectively deprecated.

Reply:
There are still some tests about FGMap. I add the manager there to maintain backward compatibility.